tests/api/onboarding-profile.php

```diff
@@ -97,7 +97,7 @@ public function test_schema() {
 		$data       = $response->get_data();
 		$properties = $data['schema']['properties'];
 
-		$this->assertCount( 11, $properties );
+		$this->assertCount( 12, $properties );
```

Comment:
Just a quick reminder that if https://github.com/woocommerce/woocommerce-admin/pull/2718 is merged first, we'll need to update this value.